### PR TITLE
Support reftest includes in Wrench

### PR DESCRIPTION
**Pull request by rlhunt**
This commit gives Wrench the ability to include reftest manifests using `include path/to/reftest.list`.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/764)

### CODE REVIEW

File: wrench/src/reftest.rs [hunk 1]
```diff
@@ -40,18 +39,28 @@ pub fn parse_reftests<F>(filename: &str, mut runner: F)
...
+        match items.next() {
+            Some("include") => {
+                let include = dir.join(items.next().unwrap());
+                parse_reftests(include.as_path(), runner);
+            }
+            Some(x) => {
+                let kind = match x {
+                    "==" => ReftestOp::Equal,
+                    "!=" => ReftestOp::NotEqual,
+                    _ => panic!(),
```
Comment (glennw) [inline wrench/src/reftest.rs]: While we're changing this, let's make this panic include a message saying unexpected match operator.

**Comment by jrmuizel** [comment]
Yeah, this looks fine to me.

**Comment by glennw** [comment]
@rlhunt OK, let's fix up the one minor issue, and then we can get this merged.

**Comment by rlhunt** [comment]
Fixed and rebased.

**Comment by glennw** [comment]
The panic message doesn't seem to have been updated?

**Comment by rlhunt** [comment]
Oh you're right. Now it should be added. Sorry!

**Comment by glennw** [comment]
Thanks!

**Pull request merged**